modules/consentManagementUsp.js

```diff
@@ -7,22 +7,19 @@
 import * as utils from '../src/utils';
 import { config } from '../src/config';
 import { uspDataHandler } from '../src/adapterManager';
-import includes from 'core-js/library/fn/array/includes';
 
-const DEFAULT_CONSENT_API = 'uspapi';
+const DEFAULT_CONSENT_API = 'iab';
```

Comment:
What is the significance of changing this to `iab`?

Reply:
we only support the `iab` api for now. That could change in the future. (e.g. we might support 'static' or even 'daa')